frontend/admin/webpack.base.js

```diff
@@ -154,8 +154,8 @@ module.exports = {
   },
   output: {
     publicPath: "/admin/", // final path for routing
-    filename: "[name].js?id=[contenthash]", // file hashing for cache busting
-    chunkFilename: "[name].js?id=[contenthash]",
+    filename: "[contenthash].[name].js", // file hashing for cache busting
+    chunkFilename: "[contenthash].[name].js", // file hashing for cache busting
```

Comment:
Don't forget the CSS hash on line 46

Reply:
Oops! I didn't even see that line!

Updated in [update css output to have hash in filename](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4988/commits/f3c65517d91453369882fef5ac140f6a969dd741)